Gemfile.lock

```diff
@@ -606,7 +606,7 @@ GEM
       rack (< 3)
     rack-test (2.1.0)
       rack (>= 1.3)
-    rackup (1.0.0)
+    rackup (1.0.1)
```

Comment:
This is going to cause issues ... either remove by hand, or re-run bundle update with `--conservative` to only bump doorkeeper.

Reply:
```suggestion
    rackup (1.0.0)
```